### ISSUE TITLE
Install fails with python 3.6

### ISSUE THREAD

**Issue by tuler** [spulec/PyQS#36]
```
Collecting pyqs
  Downloading pyqs-0.0.18.tar.gz
    Complete output from command python setup.py egg_info:
    running egg_info
    creating pip-egg-info/pyqs.egg-info
    writing pip-egg-info/pyqs.egg-info/PKG-INFO
    writing dependency_links to pip-egg-info/pyqs.egg-info/dependency_links.txt
    writing entry points to pip-egg-info/pyqs.egg-info/entry_points.txt
    writing requirements to pip-egg-info/pyqs.egg-info/requires.txt
    writing top-level names to pip-egg-info/pyqs.egg-info/top_level.txt
    writing manifest file 'pip-egg-info/pyqs.egg-info/SOURCES.txt'
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/52/1_rb04pd5597jx4cxdz5pnfc0000gn/T/pip-build-5lmk0nzk/pyqs/setup.py", line 75, in <module>
        include_package_data=True,
      File "/usr/local/Cellar/python3/3.6.0/Frameworks/Python.framework/Versions/3.6/lib/python3.6/distutils/core.py", line 148, in setup
        dist.run_commands()
      File "/usr/local/Cellar/python3/3.6.0/Frameworks/Python.framework/Versions/3.6/lib/python3.6/distutils/dist.py", line 955, in run_commands
        self.run_command(cmd)
      File "/usr/local/Cellar/python3/3.6.0/Frameworks/Python.framework/Versions/3.6/lib/python3.6/distutils/dist.py", line 974, in run_command
        cmd_obj.run()
      File "/Users/tuler/Documents/workspace/sb/v3/env/lib/python3.6/site-packages/setuptools/command/egg_info.py", line 279, in run
        self.find_sources()
      File "/Users/tuler/Documents/workspace/sb/v3/env/lib/python3.6/site-packages/setuptools/command/egg_info.py", line 306, in find_sources
        mm.run()
      File "/Users/tuler/Documents/workspace/sb/v3/env/lib/python3.6/site-packages/setuptools/command/egg_info.py", line 533, in run
        self.add_defaults()
      File "/Users/tuler/Documents/workspace/sb/v3/env/lib/python3.6/site-packages/setuptools/command/egg_info.py", line 569, in add_defaults
        sdist.add_defaults(self)
      File "/Users/tuler/Documents/workspace/sb/v3/env/lib/python3.6/site-packages/setuptools/command/py36compat.py", line 34, in add_defaults
        self._add_defaults_python()
      File "/Users/tuler/Documents/workspace/sb/v3/env/lib/python3.6/site-packages/setuptools/command/sdist.py", line 132, in _add_defaults_python
        if self.distribution.has_pure_modules():
      File "/usr/local/Cellar/python3/3.6.0/Frameworks/Python.framework/Versions/3.6/lib/python3.6/distutils/dist.py", line 980, in has_pure_modules
        return len(self.packages or self.py_modules or []) > 0
    TypeError: object of type 'filter' has no len()
```

**Comment by tuler** [comment]
Duplicates #32, sorry

**Comment by andrewgross** [comment]
Thanks, I'll investigate

**Comment by spendyala** [comment]
Is there any update on this. Still facing this issue.

**Comment by dmaterialized** [comment]
Having this issue as well with 3.6.

**Comment by smendes** [comment]
I got the same issue so I made a PR https://github.com/spulec/PyQS/pull/41 for fixing it.
There is no feature added in it, and I tried to keep it as close as possible to the current implementation without breaking compatibility with Python 2.7

**Comment by Tenzer** [comment]
Now that #41 is merged, would it be an idea to push out a new release to PyPI?

**Comment by spulec** [comment]
Fixed with #41

**Comment by spulec** [comment]
@Tenzer I've just published version `0.0.22`
